### DISCUSSION TITLE
Issue with Renew of github student

### DISCUSSION THREAD

**Discussion by SwartZCoding** [community/community#39252] [GitHub Education]
Hi all ! It's been over 3 weeks now since my school told me that my GitHub Student renewal is not possible due to a bug at Github. I feel like I'm being taken for an idiot.

Is this really a Github bug?

I contacted Github too and I've been waiting for a response for more than 2 weeks...

I find myself blocked because my school is a Github partner and I therefore cannot renew the offer myself.

**Comment by NorthBlue333** [comment]
We have a bunch of students at our school unable to verify their accounts because of some sort of a loop : click on the link provided by the school, click "Get the GitHub Student Developer Pack", redirects to "Are you a student ?". It is also not possible to "Reverify your academic affiliation" because our school is registered as a Github Partner (we must use the provided link).

We've been in this situation for over a month now, some students have successfully renewed their offers though, I do not know how. I have just opened a ticket on Github.

**Reply by SwartZCoding** [reply]
> We have a bunch of students at our school unable to verify their accounts because of some sort of a loop : click on the link provided by the school, click "Get the GitHub Student Developer Pack", redirects to "Are you a student ?". It is also not possible to "Reverify your academic affiliation" because our school is registered as a Github Partner (we must use the provided link).

Exactly same issue !!! Sort of loop... Since 2 months soon ...

**Reply by NorthBlue333** [reply]
Ok so this means it really is a bug from Github... Thanks, at least I know we are not the only ones 👍
